storagenode/storagenodedb/consoledb.go

```diff
@@ -98,7 +98,7 @@ func (db *consoledb) getDailyBandwidthUsed(ctx context.Context, cond string, arg
 	qb.WriteString("GROUP BY DATE(created_at), action ")
 	qb.WriteString("ORDER BY created_at ASC")
 
-	rows, err := db.db.QueryContext(ctx, db.Rebind(qb.String()), args)
+	rows, err := db.db.QueryContext(ctx, db.Rebind(qb.String()), args...)
 	if err != nil {
```

Comment:
This seems to be missing tests.

Reply:
let's make it a separate pull request, so the other part of the team can use the query meanwhile